ocs_ci/deployment/baremetal.py

```diff
@@ -423,6 +424,43 @@ def update_bm_status(self, bm_status):
             )
             return response.json()['message']
 
+        def create_pxe_files(self, ocp_version, role):
+            """
+            Create pxe file for giver role
+
+            Args:
+                ocp_version (str): OCP version
+                role (str): Role of node eg:- bootstrap,master,worker
+
+            Returns:
+                str: temp file path
+
+            """
+            extra_data = ""
+            if ocp_version == "4.6":
+                extra_data = f"coreos.live.rootfs_url={self.helper_node_details['bm_install_files']}" \
```

Comment:
avoid line breaks

Reply:
could also make quite a few lines shorter by storing `self.helper_node_details['bm_install_files']` in its own variable

Reply:
Done